draft-packages/hierarchical-select/docs/hierarchicalStoriesHelpers.ts

```diff
@@ -0,0 +1,134 @@
+/* eslint import/no-extraneous-dependencies: 0 */
```

Comment:
Technically, since this is a story and not something that is to be distributed, I could have linked to this file that is also present in `draft-packages/hierarchical-menu/docs/hierarchicalStoriesHelpers.ts`. It felt like code smell / something that would encourage bad practice so I have duplicated this file instead.

Reply:
Ah ok, so with this new structure, every package's stories should be self-contained?

In this case, `hierarchical-menu` uses `hierarchical-select` internally. So we're saying it's ok to references other packages for implementation code, but not for the stories.

Reply:
Yeah, pretty much. The stories are not within the distributed package so one would have to import using a relative link.